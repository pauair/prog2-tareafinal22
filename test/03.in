agregarNuevoProducto 50 prod50
agregarNuevoProducto 60 prod60
agregarNuevoProducto 70 prod70
agregarNuevoProducto 40 prod40
agregarNuevoProducto 30 prod30
agregarNuevoProducto 45 prod45
agregarNuevoProducto 55 prod55
agregarNuevoProducto 75 prod75
agregarNuevoProducto 80 prod80
agregarNuevoProducto 25 prod25
agregarNuevoProducto 10 prod10
agregarNuevoProducto 50 prod50
agregarNuevoProducto 60 prod60
agregarNuevoProducto 70 prod70
agregarNuevoProducto 40 prod40
agregarNuevoProducto 30 prod30
agregarNuevoProducto 45 prod45
agregarNuevoProducto 55 prod55
agregarNuevoProducto 75 prod75
agregarNuevoProducto 80 prod80
agregarNuevoProducto 25 prod25
agregarNuevoProducto 10 prod10
vencimientosProducto 50
vencimientosProducto 60
vencimientosProducto 70
vencimientosProducto 40
vencimientosProducto 30
vencimientosProducto 45
vencimientosProducto 55
vencimientosProducto 75
vencimientosProducto 80
vencimientosProducto 25
vencimientosProducto 10
Fin