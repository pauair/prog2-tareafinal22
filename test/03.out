1>Producto agregado 50 - prod50.
2>Producto agregado 60 - prod60.
3>Producto agregado 70 - prod70.
4>Producto agregado 40 - prod40.
5>Producto agregado 30 - prod30.
6>Producto agregado 45 - prod45.
7>Producto agregado 55 - prod55.
8>Producto agregado 75 - prod75.
9>Producto agregado 80 - prod80.
10>Producto agregado 25 - prod25.
11>Producto agregado 10 - prod10.
12>Ya existe un producto con el identificador ingresado.
13>Ya existe un producto con el identificador ingresado.
14>Ya existe un producto con el identificador ingresado.
15>Ya existe un producto con el identificador ingresado.
16>Ya existe un producto con el identificador ingresado.
17>Ya existe un producto con el identificador ingresado.
18>Ya existe un producto con el identificador ingresado.
19>Ya existe un producto con el identificador ingresado.
20>Ya existe un producto con el identificador ingresado.
21>Ya existe un producto con el identificador ingresado.
22>Ya existe un producto con el identificador ingresado.
23>Información del producto 50 - prod50
Total de unidades: 0
24>Información del producto 60 - prod60
Total de unidades: 0
25>Información del producto 70 - prod70
Total de unidades: 0
26>Información del producto 40 - prod40
Total de unidades: 0
27>Información del producto 30 - prod30
Total de unidades: 0
28>Información del producto 45 - prod45
Total de unidades: 0
29>Información del producto 55 - prod55
Total de unidades: 0
30>Información del producto 75 - prod75
Total de unidades: 0
31>Información del producto 80 - prod80
Total de unidades: 0
32>Información del producto 25 - prod25
Total de unidades: 0
33>Información del producto 10 - prod10
Total de unidades: 0
34>Fin.
